pcdsdevices/pump.py

```diff
@@ -172,6 +172,17 @@ class PIPPLC(Device):
     high_voltage_switch = Cpt(EpicsSignalWithRBV, ':HV_SW', kind='config',
                               doc='epics command to switch on the '
                               'high voltage')
+    plc_ai_offset = Cpt(EpicsSignalWithRBV, ':AI_Offset', kind='config',
+                        doc=('Analog input offset must match ion pump '
+                             'analog ouput offset. Default: 13'))
+    auto_on = Cpt(EpicsSignalWithRBV, ':Auto_On', kind='config',
+                  doc=('Ion pump with automatically turn on when '
+                       'reference gauge pressure is below protection '
+                       'setpoint'))
+    qpc_name = Cpt(EpicsSignalRO, ':VPCNAME', kind='config',
+                   doc='Quad Ion Pump Controller Name')
+    qpc_pumpsize = Cpt(EpicsSignalRO, ':PUMPSIZE', kind='config',
+                       doc='Ion Pump Size (L/s)')
```

Comment:
Is it size or max flow?

Reply:
The pump size is measured in liters per second. The reason why these items show up in the pipplc is because that is the object that is on the icon. My thought process, which may be convinced otherwise is that the name pipplc should change to a more generic one like qpcpip. The reason is because in operation, the person on the screen would like to find all relevant information for an ion pump instead of looking through the physical representation of the ion pump controller (which is a ion pump connected to a Quad Ion Pump Controller that controls 4 different ion pumps which is connected to a plc by a dsub and also by ethernet for added functionality). By having a screen that condenses all the complexity onto a screen, hopefully it will be easier to operate. The complexity is for the controls engineer to work through. Maybe there is another way to approach it.

Reply:
Interesting. Are all PLC PIPs connected through a QPC?

Reply:
Yes, all ion pumps in the FEE and EBD are connected through the QPC first. The parts of the PVs come through the plc ioc and other luxurious PVs come through another ioc called vacuum support that talks to the QPC through ethernet.

Reply:
The complexity of transducing the pressure to an analog signal is done by the controller so that the plc does not have to do all the lower level sections. On the other hand, Beckhoff is doing the lower level complexity of driving the dumb motors we have for motion